server/src/database/schema.ts

```diff
@@ -2,14 +2,16 @@ export const PreReq = `CREATE TABLE PreReq (
     CourseDept VARCHAR(255),
     CourseNumber VARCHAR(255),
     PreReqCourseDept VARCHAR(255),
-    PreReqCourseNumber VARCHAR(255)
+    PreReqCourseNumber VARCHAR(255),
+    PreReqText TEXT
```

Comment:
i used `TEXT` for this over defining a `VARCHAR(n)` because it seems like there's negligible difference between the two and i'm not sure what to set the length to otherwise

Reply:
`TEXT` or `VARCHAR` is fine here. `VARCHAR` I believe can be set to unlimited, but I think TEXT default is already unlimited.